resources/js/lang/en.js

```diff
@@ -102,17 +93,17 @@ export default {
             "name": "The Old Guard",
             "stages": {
                 0: "Jekserah has summoned an undead army and is planning to invade the city! You must warn the city guard at once!",
-                1: "Jekserah has summoned an undead army and is planning to invade the city! You must warn the city guard at once!{br}Protecting Gloomhaven with an army of undead was madness. You could not give over all the power to an unscrupulous necromancer. You ran to warn the city guard and were able to mount a defense against the waves of undead crashing against your weapons. You fought the undead all the way back to Jekserah's manor, where you put an end to her vile necromancy once and for all. Jekserah is dead by your hand.",
```

Comment:
I removed this line
>Protecting Gloomhaven with an army of undead was madness.

You've fought the undead army, you protect Gloomhaven **from** an undead army.
In my opinion the text was just as clear without this line and I prefer it a little more compact

Reply:
Again, this is a direct quote from the scenario 12 intro, and it refers to Jekserah's plan of protecting the city with undead (instead of the military).
I'd keep it.

Reply:
You're correct, I've missed it again. But it caught my attention because it seemed like the summary suggested that you as a party where Protecting Gloomhaven with an army of undead. It was untill I read the original.

> As Jekserah outlines her plan, you find yourself slowly backing out of the room. Protecting this city with an army of undead is madness.

I'll add in the detail that it's about her plan, like this:
> Her plan of protecting Gloomhaven with an army of undead was madness